networks/rpc/websocket.go

```diff
@@ -70,6 +70,12 @@ func (srv *Server) WebsocketHandler(allowedOrigins []string) http.Handler {
 		Handler: func(conn *websocket.Conn) {
 			// Create a custom encode/decode pair to enforce payload size and number encoding
 			conn.MaxPayloadBytes = common.MaxRequestContentLength
+			if WebsocketReadDeadline != 0 {
```

Comment:
The zero value is handled in `conn.SetReadDeadline` function. We don't need to check if the value is zero.

Reply:
@winnie-byun There is an addition in the next line. When `WebsocketReadDeadline == 0`, the input parameter is not zero. The next line will be the same with `conn.SetReadDeadline(time.Now())` that means "expire right now".

Reply:
I am confused with the code. If I set `WebsocketReadDeadline` to 10 seconds, does it mean I can only use the websocket for next 10 seconds after the node started?

Reply:
No, the deadline is set when each time the handler executed. It means that each client has its own deadline when the connection is established.